### ISSUE TITLE
[Help] Is it possible to check whether PTY is blocked on reading input?

### ISSUE THREAD

**Issue by Ishaangandhi** [microsoft/node-pty#443]
## Issue description

Basically, I'm interested in detecting if a spawned process is trying to read a line of input. Is that possible?

**Comment by jerch** [comment]
Nope, thats not directly possible through a PTY without further preparation like hooking up a debugger. It would need deep process introspection, thats kernel land your not meant to have access to by normal means.

**Comment by Tyriar** [comment]
Closing out of scope
